tests/conftest.py

```diff
@@ -21,6 +23,20 @@ def begin(conn):
         conn.execute('BEGIN')
 
 
+@pytest.fixture
+def event_loop(request):
```

Comment:
Why not explicitly include both pytest an pytest-asyncio in dev-requirements, so that a new tester can use requirements.txt and dev-requirements.txt to install everything he needs?